### DISCUSSION TITLE
Android Find My Device Tracker detection

### DISCUSSION THREAD

**Discussion by dirkmueller** [seemoo-lab/AirGuard#177] [Ideas]
I recently acquired a few android findmy device trackers which seem to not be detected by the application, even when only paired with a foreign phone and manually scanning.

Is it possible to detect those or does anyone know how the Android trackers are different to the apple airtag ones to start investigating where in the implementation there needs to be support added?

**Comment by Sn0wfreezeDev** [comment]
Hi Dirk, 

Thanks for reaching out. We are currently testing an update which support Google Find My Devices

**Comment by dirkmueller** [comment]
@Sn0wfreezeDev thanks for the info. How can I participate in testing?

**Comment by Sn0wfreezeDev** [comment]
Hi @dirkmueller 

We just published the newest version on GitHub: https://github.com/seemoo-lab/AirGuard/releases 

Google Play has a staged roll out. So it can take some time until you will receive an update.
